node/resolvers/index.ts

```diff
@@ -154,6 +166,63 @@ const defaultHeaders = (authToken: string) => ({
   'Proxy-Authorization': authToken,
 })
 
+// checks and configure the OrderForm based on quoteId
+const checkAndCreateQuotesConfig = async (ctx: Context): Promise<any> => {
+  const {
+    clients: { apps, checkout },
+    vtex: { logger },
+  } = ctx
+
+  const app: string = getAppId()
+  const accountSettings = await apps.getAppSettings(app)
```

Comment:
I suggest storing this in `vbase` rather than in the app settings. `vtex.b2b-quotes-graphql` is designed to be a dependency app, not directly installed, and app settings can only be used for explicitly installed apps.